eng/packaging.targets

```diff
@@ -20,6 +20,16 @@
     <UseIntellisenseDocumentationFile Condition="'$(UseIntellisenseDocumentationFile)' == '' and Exists('$(LibIntellisenseDocumentationFilePath)')">true</UseIntellisenseDocumentationFile>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(ServicingVersion)' != ''">
+    <!-- Always update the package version in servicing. -->
+    <PackageVersion>$(MajorVersion).$(MinorVersion).$(ServicingVersion)</PackageVersion>
+    <IsNetCoreApp Condition="$(NetCoreAppLibrary.Contains('$(AssemblyName);'))">true</IsNetCoreApp>
```

Comment:
I thought there was already a property to know this. We use it all over the place.

Reply:
https://github.com/dotnet/runtime/blob/22cee85aa08061e5b2c1e893797a860e3c642e6c/src/libraries/Directory.Build.targets#L42